inheritance/src/main/java/inheritance/Theater.java

```diff
@@ -0,0 +1,48 @@
+package inheritance;
+
+import java.util.ArrayList;
+
+public class Theater implements Business {
+    public String name;
+    public double stars;
+    public double price;
+    ArrayList<Object> reviews = new ArrayList<>();
+    ArrayList<String> movies = new ArrayList<>();
+
+    public Theater theaterConstructor(String name, double stars, double price) {
```

Comment:
This was discussed in Code Review in class--update this method into a constructor. 🚧